data/pages-content/finaid.md

```diff
@@ -9,31 +9,227 @@ subtitle:
+   Schengen visa to the Czech Republic (up to €80).
+
+
+## Grant Eligibility
+
+Our grants are open to everyone in need of financial support to participate in
+EuroPython. We will look at the following criteria to help decide who qualifies
+for a grant:
+
+1. **Contributors:** We want to support those who make the Python community
+   a positive, inclusive and welcoming space. We welcome grant applications
+   from folks who need financial support and who clearly put effort into
+   the community and EuroPython, including speakers, volunteers, community
+   organizers, and open-source contributors.
+2. **Social & Economic factors:** Everyone should have a chance to attend
+   EuroPython, regardless of their social or economic situation.
+3. **Diversity:** We want EuroPython to be the most diverse and inclusive event
+   possible, and we especially want to help those from under-represented groups
+   in tech feel welcomed, be present in and contribute to our community.
+
+
+## How to apply
+
+You can apply for Financial Aid by filling out the
+[application form][application-form].
+
+The data we collect on this form will exclusively be used by our Financial Aid
+team for the selection of grant recipients and the processing of financial
+aid refunds. No data is passed on to third parties, except to our accountants
+and the bank in order to process the refunds. We use Google Forms for the
+processing and Google Drive for storing the relevant information.
+
+For more details on our privacy policies, please see our
+[privacy policy][privacy-policy].
+
+
+## Timeline
+
+Grants will be awarded in two rounds. This makes it easier for us to reallocate
+grants in case grant offers were rejected in the first round. Receiving a grant
+early is also beneficial for people having to apply for a Schengen visa.
+
+If you’ve applied for the first round, you will automatically be considered
+during the second round if you did not receive a grant during the first round.
+You do not have to reapply separately for the second round. **In fact, if you
+want to increase your chances of receiving a grant, it is recommended to apply
+early.**
+
+**Note:** dates in the timeline follow the [Anywhere on Earth][AoE] convention.
+
+<table className="mt-8 finaid-timeline">
+<tbody className="text-xl">
+  <tr>
+    <td>6 March 2024</td>
+    <td></td>
+    <td>Applications open</td>
+  </tr>
+  <tr className="font-bold">
+    <td>21 April 2024</td>
+    <td>
+      <img
+        src="/img/accents/finaid-round1.svg"
+        className="findaid-accent-round-marker"
+        alt="round one" />
+    </td>
+    <td>Deadline for submitting first-round applications</td>
+  </tr>
+  <tr>
+    <td>19 May 2024</td>
+    <td>
+      <img
+        src="/img/accents/finaid-round1.svg"
+        className="findaid-accent-round-marker"
+        alt="round one" />
+    </td>
+    <td>First round of grant notifications</td>
+  </tr>
+  <tr>
+    <td>12 May 2024</td>
```

Comment:
This should happen _after_ the 19th of May, right?
How do I accept a grant if I haven't been notified I got it?

Reply:
Should be resolved here https://github.com/EuroPython/website/pull/483#discussion_r1517528292